.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+# ** patchelf isn't supported here, _yet_ **
+# addons:
+#   apt:
+#     sources:
+#       - ppa:utlemming/rax-playground
+#     packages:
+#       - build-essential
+#       - ninja-build
+#       - patchelf
+#       - libboost-dev
+#       - libboost-program-options-dev
+#       - libogg-dev
+before_install:
+    - sudo add-apt-repository ppa:utlemming/rax-playground -y
+    - sudo apt-get update -qq
+    - sudo apt-get install build-essential ninja-build libboost-dev libboost-program-options-dev libogg-dev patchelf
+install: pip install flake8 .
```

Comment:
Ah, I was wondering about that `.`. I thought it meant "install flake8 to the current directory (somehow)", not "install flake8 *and* the package located in the current directory".